view/rooms/app.js

```diff
@@ -536,6 +538,8 @@ function displayBot(bot, botIndex) {
   // We don't want to scroll the iframe, just the bot inside it
   iframe.scrolling = 'no';
   iframe.frameBorder = 0;
+  iframe.sandbox = `allow-scripts allow-same-origin
```

Comment:
@tausifmuzaffar I think strings with backticks should be something like this:

iframe.sandbox = `allow-scripts allow-same-origin\
    allow-pointer-lock allow-forms`;

to prevent a newline being inserted into the string:
ref: https://stackoverflow.com/questions/37321047/wrap-long-template-literal-line-to-multiline-without-creating-a-new-line-in-the